app/main.py

```diff
@@ -1,3 +1,15 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Callable:
+    saved_data = {}
+
+    def wrapper(*args: Any, **kwargs: Any) -> Any:
```

Comment:
args is a tuple
kwargs is a dict

Reply:
Do I need to add type annotations for args, kwargs in real projects? (they have default types)